src/main/java/com/google/sps/data/EntityConstants.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Date;
+
+public class EntityConstants {
```

Comment:
for these constant classes, people also privatize the constructor so that the class isn't instantiated

Reply:
private EntityConstants() {}